index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MERCY MUGAMBI</title>
+
+    <!--stylesheet-->
+    <link rel="stylesheet" href="/style.css" />
+  </head>
+  <body>
+    <header class="header-nav">
+      <div class="logo">MER<span>cy</span></div>
+
+      <nav class="hamburger">
+        <img src="/assets/images/bars.png" />
+      </nav>
+    </header>
+
+    <main class="heading">
```

Comment:
- [ ] Kindly wrap your headline section in an `<section>` tag. It represents a generic standalone section of the document.[Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.